config/plugin/network_problem.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This plugin checks for common network issues. Currently, it only checks
+# if the conntrack table is full. 
+
```

Comment:
I would prefer we keep the plugin pattern for defining exit code. Which is illustrated in [check_ntp.sh](https://github.com/kubernetes/node-problem-detector/blob/master/config/plugin/check_ntp.sh#L6-L8)
```
OK=0
NONOK=1
UNKNOWN=2
```

Reply:
Done